### DISCUSSION TITLE
Connector and ForgifiedFabricAPI for the server

### DISCUSSION THREAD

**Discussion by MinecraftDeishu** [Sinytra/Connector#320] [Q&A]
Crashes the server, only these two mods added, will in the future be able to install these two mods for the server or perhaps I do something wrong ?

**Comment by Su5eD** [comment]
Please provide more information: what mods do you have installed?

**Reply by MinecraftDeishu** [reply]
I just tested to run the server with Connector and Forgified Fabric API, the core for the server used either ArcLight or Mohist, on both crashes, if I'm not mistaken because of Forgified Fabric API, there are no other mods in general no more, only these two put.

**Reply by Su5eD** [reply]
Try using the official vanilla server

**Reply by MinecraftDeishu** [reply]
Ah, I see, you may not have understood what I mean, it's not minecraft that crashes, it's the server that crashes on startup, the server with these two mods won't start, as the server does not start and crashes, I can write that writes in crash report.

**Reply by Su5eD** [reply]
Hybrids like ArcLight and Mohist are not supported, use the official vanilla server

**Reply by MinecraftDeishu** [reply]
1. But on vanilla minecraft server can not put mods.
2. will there be support for mohist/arclight in the future ? 
3. Here is such an error on minecraft vanilla forge server, that is the server that I installed through the official installer forge, also tried neoforge, the same error, as you can see on this error says that I need forge 47.1.3 but I already have 47.1.3, so what is the problem ?
Mod ID: 'forge', Requested by: 'fabric_convention_tags_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_transitive_access_wideners_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_command_api_v2', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_lifecycle_events_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_registry_sync_v0', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_game_rule_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_events_interaction_v0', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_models_v0', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_content_registries_v0', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_resource_conditions_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_rendering_data_attachment_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_api_base', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_rendering_fluids_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_blockrenderlayer_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_renderer_indigo', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_dimensions_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_item_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_object_builder_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_block_view_api_v2', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_model_loading_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_biome_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_rendering_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_screen_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_transfer_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_api_lookup_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_key_binding_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_loot_api_v2', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_resource_loader_v0', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_renderer_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_recipe_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_api', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_networking_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_data_generation_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_particles_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_block_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_message_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_entity_events_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_screen_handler_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_sound_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_mining_level_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_item_group_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'
        Mod ID: 'forge', Requested by: 'fabric_client_tags_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.30'

**Reply by saodMinecarft01** [reply]
bro you're using 47.1.30 not 47.1.3

**Reply by TouseefX** [reply]
minecraft servers don't have 47.1.3

**Comment by TouseefX** [comment]
well my works but the server just crashes cuz i don't have same mods at my client folder
